reduce-practice.js

```diff
@@ -1,235 +1,235 @@
+          "Language": "English, Mandarin",
+          "Country": "USA, UK",
+          "Awards": "Won 2 Oscars. Another 146 wins & 142 nominations.",
+          "Poster": "http://ia.media-imdb.com/images/M/MV5BMTMxNTMwODM0NF5BMl5BanBnXkFtZTcwODAyMTk2Mw@@._V1_SX300.jpg",
+          "Metascore": "82",
+          "imdbRating": "9.0",
+          "imdbVotes": "1,652,832",
+          "imdbID": "tt0468569",
+          "Type": "movie",
+          "Response": "True"
+       },
+       {  
+          "Title": "Batman Begins",
+          "Year": "2005",
+          "Rated": "PG-13",
+          "Released": "15 Jun 2005",
+          "Runtime": "140 min",
+          "Genre": "Action, Adventure",
+          "Director": "Christopher Nolan",
+          "Writer": "Bob Kane (characters), David S. Goyer (story), Christopher Nolan (screenplay), David S. Goyer (screenplay)",
+          "Actors": "Christian Bale, Michael Caine, Liam Neeson, Katie Holmes",
+          "Plot": "After training with his mentor, Batman begins his fight to free crime-ridden Gotham City from the corruption that Scarecrow and the League of Shadows have cast upon it.",
+          "Language": "English, Urdu, Mandarin",
+          "Country": "USA, UK",
+          "Awards": "Nominated for 1 Oscar. Another 15 wins & 66 nominations.",
+          "Poster": "http://ia.media-imdb.com/images/M/MV5BNTM3OTc0MzM2OV5BMl5BanBnXkFtZTYwNzUwMTI3._V1_SX300.jpg",
+          "Metascore": "70",
+          "imdbRating": "8.3",
+          "imdbVotes": "972,584",
+          "imdbID": "tt0372784",
+          "Type": "movie",
+          "Response": "True"
+       },
+       {
+          "Title": "Avatar",
+          "Year": "2009",
+          "Rated": "PG-13",
+          "Released": "18 Dec 2009",
+          "Runtime": "162 min",
+          "Genre": "Action, Adventure, Fantasy",
+          "Director": "James Cameron",
+          "Writer": "James Cameron",
+          "Actors": "Sam Worthington, Zoe Saldana, Sigourney Weaver, Stephen Lang",
+          "Plot": "A paraplegic marine dispatched to the moon Pandora on a unique mission becomes torn between following his orders and protecting the world he feels is his home.",
+          "Language": "English, Spanish",
+          "Country": "USA, UK",
+          "Awards": "Won 3 Oscars. Another 80 wins & 121 nominations.",
+          "Poster": "http://ia.media-imdb.com/images/M/MV5BMTYwOTEwNjAzMl5BMl5BanBnXkFtZTcwODc5MTUwMw@@._V1_SX300.jpg",
+          "Metascore": "83",
+          "imdbRating": "7.9",
+          "imdbVotes": "876,575",
+          "imdbID": "tt0499549",
+          "Type": "movie",
+          "Response": "True"
+       }
+    ];
+
+    // Use reduce to find the average IMDB rating of the movies directed by Christopher Nolan.
+
+    // method 1
+    const filteredMovies = watchList.filter(movie => (movie.Director === "Christopher Nolan"));
```

Comment:
This is great
You don't actually need the extra parens inside the filter function:
`.filter(movie => movie.director === "Christofer Nolan")